README.md

```diff
@@ -129,6 +130,14 @@ Here are all the custom parameters in their glory:
     <a class="pinterest" data-media="https://picsum.photos/500">Pin</a>
     <a class="telegram" data-text="Check this out!"></a>
     <a class="twitter" data-via="MyNickname"></a>
+    <a class="mastodon" data-text="Check this out!" data-via="@MyNickname@myserver.social"></a>
     <a class="whatsapp">Send</a>
 </div>
 ```
+
+## Copyright
```

Comment:
This is a brilliant thing that I forgot — logo copyrights! Thank you for pointing this out. I'll add an issue regarding that

Reply:
Ah cool, thanks!